plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -1753,4 +1756,345 @@ private Set<String> getAllFilesFromCdfDirectory(String tableName)
+                new TrinoSqlExecutor(getQueryRunner(), session),
+                "test_partition_filter_included",
+                "(x VARCHAR, part INT) WITH (PARTITIONED_BY = ARRAY['part'])",
+                ImmutableList.of("('a', 1)", "('a', 2)", "('a', 3)", "('a', 4)", "('b', 1)", "('b', 2)", "('b', 3)", "('b', 4)"))) {
+            assertQuery(session, "SELECT * FROM " + table.getName() + " WHERE part = 1", "VALUES ('a', 1), ('b', 1)");
+            assertQuery(session, "SELECT count(*) FROM " + table.getName() + " WHERE part < 2", "VALUES 2");
+            assertQuery(session, "SELECT count(*) FROM " + table.getName() + " WHERE Part < 2", "VALUES 2");
+            assertQuery(session, "SELECT count(*) FROM " + table.getName() + " WHERE PART < 2", "VALUES 2");
+            assertQuery(session, "SELECT count(*) FROM " + table.getName() + " WHERE parT < 2", "VALUES 2");
+            assertQuery(session, "SELECT count(*) FROM " + table.getName() + " WHERE part % 2 = 0", "VALUES 4");
+            assertQuery(session, "SELECT count(*) FROM " + table.getName() + " WHERE part - 2 = 0", "VALUES 2");
+            assertQuery(session, "SELECT count(*) FROM " + table.getName() + " WHERE part * 4 = 4", "VALUES 2");
+            assertQuery(session, "SELECT count(*) FROM " + table.getName() + " WHERE part % 2 > 0", "VALUES 4");
+            assertQuery(session, "SELECT count(*) FROM " + table.getName() + " WHERE part % 2 = 1 and part IS NOT NULL", "VALUES 4");
+            assertQuery(session, "SELECT count(*) FROM " + table.getName() + " WHERE part IS NULL", "VALUES 0");
+            assertQuery(session, "SELECT count(*) FROM " + table.getName() + " WHERE part = 1 OR x = 'a' ", "VALUES 5");
+            assertQuery(session, "SELECT count(*) FROM " + table.getName() + " WHERE part = 1 AND  x = 'a' ", "VALUES 1");
+            assertQuery(session, "SELECT x, count(*) AS COUNT FROM " + table.getName() + " WHERE part > 2 GROUP BY x ", "VALUES ('a', 2), ('b', 2)");
+            assertQueryFails(session, "SELECT count(*) FROM " + table.getName() + " WHERE x= 'a'", "Filter required on .*" + table.getName() + " for at least one partition column:.*");
+        }
+    }
+
+    @Test(dataProvider = "columnNamesCaseSensitive")
+    public void testPartitionFilterIncludeCaseSensitivity(String columnName)
+    {
+        Session session = sessionWithPartitionFilterRequirement();
+        try (TestTable table = new TestTable(
+                new TrinoSqlExecutor(getQueryRunner(), session),
+                "test_partition_filter_included",
+                "(x VARCHAR, " + columnName + " INT) WITH (PARTITIONED_BY = ARRAY['" + columnName + "'])",
+                ImmutableList.of("('a', 1)", "('a', 2)"))) {
+            for (String partitionColumn : columnNamesCaseSensitive()) {
+                assertQuery(session, "SELECT * FROM " + table.getName() + " WHERE " + partitionColumn + " = 1", "VALUES ('a', 1)");
+            }
+        }
+    }
```

Comment:
This test doesn't create a table with uppercase column name characters.

Reply:
I've just realized that Trino does the autodowncase of the tablename, let me remove parameterization from CT and just leave it for the query assertion

Reply:
The new change doesn't make sense. Please register the table from existing resource or use product test for creating a table having uppercase column names.

Reply:
I created a testcase within product test.